web/src/styles/index.scss

```diff
@@ -13,3 +13,7 @@
 @import '/scss/password-meter';
 @import '/scss/print';
 @import '/scss/section';
+
+h1.apd--title {
+  margin-left: 72px;
+}
```

Comment:
In my open PR, I added a `type.scss` file. This would be right at home there -- want to go ahead and create it in your PR also?

Reply:
Done!